format/index.md

```diff
@@ -887,8 +873,8 @@ a relationship link URL that does not exist.
 does not exist. For example, when `/articles/1` does not exist, request to
 `/articles/1/relationships/tags` returns `404 Not Found`.
 
-If a relationship link URL exists but the relationship is empty, then
-`200 OK` **MUST** be returned, as described above.
+> Note: If a relationship link URL exists but the relationship is empty, then
+`200 OK` will be returned, as described above.
```

Comment:
I'm good with making this non-normative so that the normative statement is in one place :+1: